packages/template-retail-react-app/app/components/swatch-group/swatch.jsx

```diff
@@ -63,10 +63,8 @@ const Swatch = ({
             aria-checked={selected}
             variant="outline"
             role="radio"
-            // To mimic the behavior of native radio inputs, only one input should be focusable.
-            // (The rest are selectable via arrow keys.)
-            tabIndex={isFocusable ? 0 : -1}
-            {...(href ? {} : selectHandlers)}
+            {...(isFocusable ? {tabIndex: 0} : {})}
+            {...(href && !handleSelect ? {} : selectHandlers)}
```

Comment:
```suggestion
            {...(href ? {} : selectHandlers)}
```

Can we double check this condition please? Currently it's possible for the `selectHandlers` to be used if there's no href. I think that's not what we wanted.

My vote is to revert to the original code. We don't need to check for `handleSelect` here because it's already been done in this code block: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/f767f58117924881cef40b33c6fa64b991aa670a/packages/template-retail-react-app/app/components/swatch-group/swatch.jsx#L48-L50